Quaver.Shared/Screens/Select/UI/Search/MapsetSearchContainer.cs

```diff
@@ -88,6 +88,8 @@ public MapsetSearchContainer(SelectScreenView view)
             CreateOrderByDateAddedButton();
             CreateTextMapsetsFound();
 
+            SearchBox.RawText = SelectScreen.PreviousSearchTerm;
```

Comment:
This needs to be set when [initializing the object](https://github.com/Quaver/Quaver/blob/5c606d7d22c5a4d0cbca85612dcd576c36287faf/Quaver.Shared/Screens/Select/UI/Search/MapsetSearchContainer.cs#L153).